README.md

```diff
@@ -1,13 +1,91 @@
-Welcome to Inspect Flow. Inspect Flow is a workflow stack built on Inspect AI that enables research organizations to run AI evaluations at scale.
+# Inspect Flow
 
-## Installation
+Workflow orchestration for [Inspect AI](https://inspect.aisi.org.uk/) that enables you to run evaluations at scale with reproducibility and maintainability.
 
-Latest development version:
+## Why Inspect Flow?
+
+As evaluation workflows grow in complexity—running multiple tasks across different models with varying parameters—managing these experiments becomes challenging. Inspect Flow addresses this by providing:
+
+- **Declarative Configuration**: Define your entire evaluation pipeline in type-safe Pydantic schemas
+- **Reproducibility**: Encapsulated Python dependencies for each workflow
+- **Parameter Sweeping**: Matrix execution patterns for systematic exploration across tasks and models
+
+Inspect Flow is designed for researchers and engineers running systematic AI evaluations who need to scale beyond ad-hoc scripts.
+
+## Quick Start
+
+### Installation
 
 ```bash
 pip install git+https://github.com/meridianlabs-ai/inspect_flow
 ```
 
+### Your First Evaluation
+
+Create a `config.py` file:
```

Comment:
Can we create these files in examples and then reference inline:
```python
--8<-- "examples/matrix_config.py"
```
Then we can add unit tests to make sure that they parse correctly and stay up to date as we make other changes.